internal/helpers/pagination_helper.go

```diff
@@ -35,3 +37,33 @@ func (p PaginationHelper[T]) DecodeCursor(cursor string) (*T, error) {
 
 	return &res, nil
 }
+
+func (p PaginationHelper[T]) ValidateFirstLast(first, last *int, maxPageSize int) (*int, error) {
```

Comment:
You are right, will remove it as part of the Pagination helper, it can be a standalone function in thesame file. However, pagination helper has a generic type, because it does just that, provides a generic function to help return a cursor of any type.